### PR TITLE
Issue #2086 - Change default bulkdata directory

### PR DESCRIPTION
**Pull request by tbieste**
Issue #2086 - Change default bulkdata directory

Signed-off-by: Troy Biesterfeld <tbieste@us.ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -2071,7 +2071,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/storageProviders/<source>/location`|string|Object store location |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointInternal`|string|Object store end point url used to read/write from COS |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointExternal`|string|Object store end point url used in the constructed download URLs|
-|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory |
+|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory. It may be desirable to set this to a path within a mount point of separate file system if large amounts of data are involved|
```
Comment (prb112) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:2074]:
```suggestion
|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory. It is recommended not to mount this to the root file system or the share a direct volume mount.  For instance, if the volume is /output/bulkdata, use /output/bulkdata/data to ensure a failed mount does not result in writing to the root file system.|
```

**Comment by prb112** [comment]
Looks good, when it passes, I'll mark LGTM

File: docs/src/pages/guides/FHIRServerUsersGuide.md (hunk 1 above)
Comment (tbieste) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: Updated.

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 2]
```diff
@@ -2071,7 +2071,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/bulkdata/storageProviders/<source>/location`|string|Object store location |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointInternal`|string|Object store end point url used to read/write from COS |
 |`fhirServer/bulkdata/storageProviders/<source>/endpointExternal`|string|Object store end point url used in the constructed download URLs|
-|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory |
+|`fhirServer/bulkdata/storageProviders/<source>/fileBase`|string| The absolute path of the output directory. It is recommended not to mount this to the root file system or the share a direct volume mount. For instance, if the volume is /output/bulkdata, use /output/bulkdata/data to ensure a failed mount does not result in writing to the root file system.|
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:2074]: "or the share a direct volume mount" is awkward

Comment (tbieste) [reply docs/src/pages/guides/FHIRServerUsersGuide.md]: Updated to hopefully less awkward, but still valid, wording.

**Review by lmsurpre — APPROVED** [review]
LGTM

**Review by prb112 — APPROVED** [review]

**Pull request merged**